src/main/java/com/urswolfer/gerrit/client/rest/http/changes/RevisionApiRestClient.java

```diff
@@ -160,6 +165,13 @@ public MergeableInfo mergeable() throws RestApiException {
         return comments("drafts");
     }
 
+    @Override
+    public Set<String> reviewed() throws RestApiException {
+        String request = getRequestPath() + "/files?reviewed";
+        JsonElement jsonElement = gerritRestClient.getRequest(request);
+        return new HashSet<>(reviewInfoParser.parseFileInfos(jsonElement));
```

Comment:
We could return a `LinkedHashSet` here, in order to prevent ordering returned by the API (not sure if it is relevant, but it also should not hurt). Also, you might want to return the correct type right from the parser?